core/src/avm2/globals/flash/display/Screen.as

```diff
@@ -0,0 +1,66 @@
+package flash.display
+{
+  import __ruffle__.stub_getter;
+  import flash.geom.Rectangle;
+
+  [API("661")]
+  public final class Screen
+  {
+    private static var _screens:Array;
+    private var _width:int;
+    private var _height:int;
+
+    [API(661)]
+    public var mode:ScreenMode;
+
+    public function Screen(width:int, height:int)
+    {
+      _width = width;
+      _height = height;
+      mode = new ScreenMode(width, height);
+    }
+
+    public static function get mainScreen():Screen
+    {
+      return screens[0];
+    }
+
+    public static native function getScreensInternal():Array;
```

Comment:
1. I believe this method shouldn't be public.
2. Currently this method is being executed at most once, but the screen configuration may change in time. I'd go with making all methods native and reading always the most recent info (assuming it's what AIR does).